
public class CheckedTest2 {
	public static void main(String[] args){
		throw new Exception("An error occured.");
	}
}
